deploy/config

```diff
@@ -4,7 +4,7 @@ port 8080;   # This is also a comment.
 
 threads 4;
 
-path /static1 StaticHandler {
```

Comment:
I don't think we should get rid of this static1 handler because I think some of our unit tests rely on it

Also is the config file in the root directory doing anything or can it be deleted?